modules/dr-restoring-cluster-state.adoc

```diff
@@ -296,7 +296,12 @@ Perform the following step only if you are using `OVNKubernetes` network plugin.
 ====
 +
 . Restart the Open Virtual Network (OVN) Kubernetes pods on all the hosts.
-
++
+[NOTE]
+====
+Validating and mutating admission webhooks may reject pods. If you add any additional webhooks and the `failurePolicy` is set to `Fail` in those webhooks, then they may reject pods. As a result, the restoration process may fail. You can avoid this failure by disabling that is by saving and deleting those webhooks while restoring the cluster state. After the cluster state is restored successfully, you can enable that is recreate the webhooks again. Alternatively, you can temporarily set the `failurePolicy` to `Ignore` while restoring the cluster state. After the cluster state is restored successfully, you can set the `failurePolicy` to `Fail`.
```

Comment:
IBM style says to use "may" with caution. I would suggest using "can" or "might" as the style guide suggests. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m (see the word "may" under the M's)

Reply:
Addressed